client/src/reducers/meals_reducer.js

```diff
@@ -0,0 +1,12 @@
+import { CREATE_MEAL } from '../actions';
+
+let initialState = {};
+
+export default function reducer(state = initialState, action) {
+  switch (action.type) {
+  	case (CREATE_MEAL) :
+  	  return state;
+  	default :
+  		return state;
```

Comment:
Let's fix the indentation here. I thought `prettier` would catch this, but I guess it didn't.